containers/hugging-face-inference/README.md

```diff
@@ -0,0 +1,33 @@
+# Hugging Face Models
+
```

Comment:
**issue(docs):** this README has a lot of good technical detail, but no high-level explanation of what the example does. We need to explain what the example does, what SCW resources it uses, and link to Hugging Face and the models used (and any interesting Python libraries we use too).